src/containers/Convert.jsx

```diff
@@ -1,71 +1,91 @@
-          <Button variant="contained" component="label" sx={{ mb: '10px' }}>
-            Upload CSV File
-            <input hidden accept=".csv" type="file" onChange={handleUploadFileChange} />
-          </Button>
-          {file && <Typography variant="h5">{file.name}</Typography>}
-        </Box>
-      </Box>
-      <Box display="flex" flexDirection="column" margin="10px">
-        <Button onClick={handleCsvToJson}>===={'>'}</Button>
-        <Button onClick={handleJsonToCsv}>{'<'}====</Button>
-      </Box>
-      <Box
-        border="solid 4px white"
-        borderRadius="8px"
-        height="400px"
-        width="300px"
-        padding="8px"
-        display="flex"
-        flexDirection="column"
-      >
-        <Box>
+    <Box>
+      <Box display="flex" flexDirection="row" alignItems="center">
+        <Box
+          display="flex"
+          flexDirection="column"
+          border="solid 4px white"
+          borderRadius="12px"
+          height="400px"
+          width="400px"
+          padding="8px"
+        >
           <Typography variant="h4" sx={{ margin: '8px' }}>
-            Json
+            CSV
           </Typography>
+          <Box display="flex" flex="1" flexDirection="column" alignItems="center" justifyContent="center">
+            {file && (
+              <Typography variant="h5" sx={{ width: '100%', overflow: 'visible' }}>
+                {file.name}
+              </Typography>
+            )}
+            <TextField
+              multiline
+              rows={8}
+              value={csvText}
+              onChange={handleCsvChange}
+              sx={{ bgcolor: 'white', width: '100%' }}
+            />
+          </Box>
         </Box>
-        <Box flex="1 1 0" sx={{ overflowY: 'scroll' }}>
-          {isConvertToJson ? (
-            <>
-              <Box textAlign="left">
-                <Typography variant="body1">[</Typography>
-              </Box>
-              {jsonData.map((obj) => (
-                <Box key={JSON.stringify(obj)} textAlign="left" pl="1rem" sx={{ wordWrap: 'break-word' }}>
-                  <Typography variant="body1">{JSON.stringify(obj)},</Typography>
-                </Box>
-              ))}
-              <Box textAlign="left">
-                <Typography variant="body1">]</Typography>
-              </Box>
-            </>
-          ) : (
-            <Box textAlign="left">
-              <Typography variant="body1">[]</Typography>
-            </Box>
-          )}
+        <Box display="flex" flexDirection="column" margin="10px">
+          <Button onClick={handleCsvToJson}>===={'>'}</Button>
+          <Button onClick={handleJsonToCsv}>{'<'}====</Button>
         </Box>
+        <Box
+          border="solid 4px white"
+          borderRadius="8px"
+          height="400px"
+          width="400px"
+          padding="8px"
+          display="flex"
+          flexDirection="column"
```

Comment:
inline styling is quick, but what would you use if you want to reuse the same style and you don't want to repeat yourself?